projects/js-packages/components/components/dialog/stories/declaration.d.ts

```diff
@@ -0,0 +1 @@
+declare module '*.module.scss';
```

Comment:
Do mean this instead?
```suggestion
declare module '*.png';
```